packages/grafana-schema/src/common/plugin.cue

```diff
@@ -0,0 +1,122 @@
+package common
+
+// TODO docs
+PluginMeta: {
```

Comment:
given how big the potential data_frame/meta issue will be, can this be its own project?

Reply:
Yeah, ok, I'll move it to a separate PR since those types aren't 100% usable in the current state anyways.